tests/auth_tests/test_management.py

```diff
@@ -183,6 +196,16 @@ def test_that_changepassword_command_with_database_option_uses_given_db(self, mo
 )
 class CreatesuperuserManagementCommandTestCase(TestCase):
 
+    def test_no_email_argument(self):
+        new_io = StringIO()
+        with self.assertRaisesMessage(CommandError, 'You must use --email with --noinput.'):
+            call_command(
```

Comment:
Single line looks fine here, otherwise you'd wantto include a trailing comma on the last line.